kernel/src/table_changes/scan.rs

```diff
@@ -234,9 +234,9 @@ mod tests {
             vec![
                 ColumnType::Selected("part".to_string()),
                 ColumnType::Selected("id".to_string()),
-                ColumnType::Selected("_change_type".to_string()),
-                ColumnType::Selected("_commit_version".to_string()),
-                ColumnType::Selected("_commit_timestamp".to_string()),
+                ColumnType::Selected(CHANGE_TYPE_COL_NAME.to_string()),
+                ColumnType::Selected(COMMIT_VERSION_COL_NAME.to_string()),
+                ColumnType::Selected(COMMIT_TIMESTAMP_COL_NAME.to_string()),
```

Comment:
actually (and sorry for the churn) I think I like the literal strings here: this let's us check that they are the actual strings we expect in the protocol (failure case we prevent is someone changes the const and then this test would change and pass)

Reply:
aha that makes sense. So we get to assert more about the code through this test 👍